ddtrace/internal/writer.py

```diff
@@ -250,7 +251,8 @@ def __init__(
         max_payload_size=None,  # type: Optional[int]
         timeout=agent.get_trace_agent_timeout(),  # type: float
         dogstatsd=None,  # type: Optional[DogStatsd]
-        report_metrics=False,  # type: bool
+        report_health_metrics=False,  # type: bool
+        report_telemetry_metrics=True,  # type: bool
```

Comment:
```suggestion
```

Reply:
If we decide to keep this argument the default value should be False. This feature should only be enabled using the globs config.